app/Resources/views/index.html.twig

```diff
@@ -9,8 +9,9 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>mozilla.ch</title>
-    <meta name="description" content="">
+    <title>Mozilla Switzerland</title>
```

Comment:
If we would set the title to something language specific, it should be localizeable. We settled for mozilla.ch since that is language independent.